deeppavlov/configs/squad/squad.json

```diff
@@ -0,0 +1,100 @@
+{
+  "dataset_reader": {
+    "name": "squad_dataset_reader",
+    "data_path": "squad/"
+  },
+  "dataset_iterator": {
+    "name": "squad_iterator",
+    "seed": 1337,
+    "shuffle": true
+  },
+  "chainer": {
+    "in": ["c_raw", "q_raw"],
```

Comment:
Maybe rename it to `context` and `question`? Interact would look better.

Reply:
Agree. Meaningful names are much better.

Reply:
renamed